src/calculateRentalCost.js

```diff
@@ -29,6 +29,19 @@
 
 function calculateRentalCost(days) {
   // write code here
+  let totalAmount = 0;
+
+  for (let i = 0; i < days; i++) {
+    totalAmount += 40;
+  }
```

Comment:
maybe it makes sense to put day cost and discounts into variables?)
https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md